packages/cluster/src/common.js

```diff
@@ -1,23 +1,32 @@
-const { reporter } = require('@dhis2/cli-helpers-engine')
+const path = require('path')
+const fs = require('fs')
 
-const dockerComposeCacheName = 'd2-cluster-docker-compose-v2'
+const { Cache, reporter } = require('@dhis2/cli-helpers-engine')
+
+const dockerComposeCacheName = 'docker-compose'
 
 module.exports.initDockerComposeCache = async ({
+    composeProjectName,
     cache,
     dockerComposeRepository,
     force,
 }) => {
-    const exists = await cache.exists(dockerComposeCacheName)
+    // make new cache dir per cluster in the root cache
+    const clusterCache = new Cache({
```

Comment:
It should be possible to cache and lookup files with deep paths using the main cache, actually, so I don't think this multi-layered cache is necessary?  You should be able to just do `cache.exists(path.join(composeProjectName, dockerComposeCacheName))`

Reply:
I'll give it a whack.

Reply:
The reason I'm using the multi-layer cache is actually to get it to create the sub-directory for me. Just `cache.exists()` just checks for it, it doesn't guarantee that it is there.

Reply:
What we have now:
```
ls -1 ~/.cache/d2/cache/
cluster-db-dev.sql.gz
d2-cluster-docker-compose-v2
```

I was going for something like:

- ~/.cache/d2/cache/{cluster name}/docker-compose/docker-compose.yml`
- ~/.cache/d2/cache/{cluster name}/clusterConfig.json`